src/node-lib/curriculum-api-2023/queries/lessonOverview/lessonOverview.query.ts

```diff
@@ -197,6 +197,8 @@ export const transformedLessonOverviewData = (
     lessonGuideUrl: content.lessonGuideAssetObjectUrl ?? null,
     phonicsOutcome: content.phonicsOutcome,
     pathways: pathways,
+    hasMediaClips: false,
```

Comment:
defaulted to false so the media clips don't appear on the non beta lesson overview pages